### ISSUE TITLE
Path separator issue on Windows

### ISSUE THREAD

**Issue by TimYorke** [asny/three-d-asset#11]
Running the three-d "lights" example results in the following error on Windows:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: NotLoaded("examples/assets/sponza\\11872827283454512094.jpg")', examples/lights/src/main.rs:109:51
```

I traced it to ```three_d_asset::io::raw_assets::RawAssets::match_path``` which ends up trying and failing to match ```"examples/assets/sponza/11872827283454512094.jpeg"``` stored in the HashMap with ```"examples/assets/sponza\11872827283454512094.jp"``` (note the incorrect separator)

**Comment by asny** [comment]
Thanks for reporting! 👍 I have committed a potential to fix it in e541226, can I get you to test it (I don't have access to a windows computer)? You'll just need to replace the `three-d-asset` dependencies in the `Cargo.toml` file in `three-d` with

```
three-d-asset = {git = "https://github.com/asny/three-d-asset", branch = "0.2"}
```

**Comment by TimYorke** [comment]
The fix looks good.

![image](https://user-images.githubusercontent.com/9195622/176147463-cbadb370-f3ff-450b-ae1c-80131ace482b.png)

**Comment by asny** [comment]
Thanks 🙏 and thanks again for reporting 🙂

**Comment by asny** [comment]
FYI, I also fixed this in the 0.1 branch and made a release (0.1.1) so it's fixed in three-d 0.12 by doing a `cargo update`
